cc.spek.Spek.yml

```diff
@@ -0,0 +1,227 @@
+      - --enable-decoder=mp2
+      - --enable-decoder=mp2float
+      - --enable-decoder=mp3
+      - --enable-decoder=mp3adu
+      - --enable-decoder=mp3adufloat
+      - --enable-decoder=mp3float
+      - --enable-decoder=mp3on4
+      - --enable-decoder=mp3on4float
+      - --enable-decoder=mpc7
+      - --enable-decoder=mpc8
+      - --enable-decoder=opus
+      - --enable-decoder=ra_144
+      - --enable-decoder=ra_288
+      - --enable-decoder=ralf
+      - --enable-decoder=shorten
+      - --enable-decoder=tak
+      - --enable-decoder=tta
+      - --enable-decoder=vorbis
+      - --enable-decoder=wavpack
+      - --enable-decoder=wmalossless
+      - --enable-decoder=wmapro
+      - --enable-decoder=wmav1
+      - --enable-decoder=wmav2
+      - --enable-decoder=wmavoice
+
+      - --enable-decoder=pcm_alaw
+      - --enable-decoder=pcm_bluray
+      - --enable-decoder=pcm_dvd
+      - --enable-decoder=pcm_f32be
+      - --enable-decoder=pcm_f32le
+      - --enable-decoder=pcm_f64be
+      - --enable-decoder=pcm_f64le
+      - --enable-decoder=pcm_lxf
+      - --enable-decoder=pcm_mulaw
+      - --enable-decoder=pcm_s8
+      - --enable-decoder=pcm_s8_planar
+      - --enable-decoder=pcm_s16be
+      - --enable-decoder=pcm_s16be_planar
+      - --enable-decoder=pcm_s16le
+      - --enable-decoder=pcm_s16le_planar
+      - --enable-decoder=pcm_s24be
+      - --enable-decoder=pcm_s24daud
+      - --enable-decoder=pcm_s24le
+      - --enable-decoder=pcm_s24le_planar
+      - --enable-decoder=pcm_s32be
+      - --enable-decoder=pcm_s32le
+      - --enable-decoder=pcm_s32le_planar
+      - --enable-decoder=pcm_u8
+      - --enable-decoder=pcm_u16be
+      - --enable-decoder=pcm_u16le
+      - --enable-decoder=pcm_u24be
+      - --enable-decoder=pcm_u24le
+      - --enable-decoder=pcm_u32be
+      - --enable-decoder=pcm_u32le
+      - --enable-decoder=dsd_lsbf
+      - --enable-decoder=dsd_msbf
+      - --enable-decoder=dsd_lsbf_planar
+      - --enable-decoder=dsd_msbf_planar
+
+      - --enable-parser=aac
+      - --enable-parser=aac_latm
+      - --enable-parser=ac3
+      - --enable-parser=cook
+      - --enable-parser=dca
+      - --enable-parser=flac
+      - --enable-parser=gsm
+      - --enable-parser=mpegaudio
+      - --enable-parser=tak
+      - --enable-parser=vorbis
+    cleanup:
+      - /share/ffmpeg/examples
+    sources:
+      - type: archive
+        url: https://ffmpeg.org/releases/ffmpeg-5.1.2.tar.xz
+        sha256: 619e706d662c8420859832ddc259cd4d4096a48a2ce1eefd052db9e440eef3dc
+        x-checker-data:
+          type: anitya
+          project-id: 5405
+          url-template: https://ffmpeg.org/releases/ffmpeg-$version.tar.xz
+
+  - name: spek
+    buildsystem: autotools
+    post-install:
+      # logo with name spek is needed but will be renamed by rename-icon -> rename it here and symlink it
+      - mv /app/share/icons/hicolor/scalable/apps/spek.svg /app/share/icons/hicolor/scalable/apps/cc.spek.Spek.svg
```

Comment:
just add `copy-icon: true` next to `rename-icon`.

Reply:
Thanks! I should have known there is something for that and search the docs.